pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Description
+*A clear and concise description of what the pull request is about.*
+
+# Type of changes
+- [ ] Bugfix / resolves *#issue_number*
```

Comment:
Perhaps we can move this `resolves *#issue_number*` to the description section? When I read `Bugfix / resolves *#issue_number*` I get the impression that only bugfixes should be linked to issues when, ideally, all PR's should be linked to an open issue.

Reply:
sure!